cdk/queues.py

```diff
@@ -51,6 +51,7 @@ def add_monitoring(self, monitoring):
             metric=self.upload_queue.metric("ApproximateNumberOfMessagesVisible"),
             statistic="sum",
             threshold=40,
+            evaluation_periods=1,
             period=core.Duration.minutes(5),
```

Comment:
evaluation_periods was missing and is required

Reply:
yep got it